mq-maintenance.fonq.nl/controller/Message/Overview.php

```diff
@@ -110,6 +110,34 @@ function doRequeue()
         $this->redirect($return_url);
         exit();
     }
+
+    function doDeadLetter()
```

Comment:
So you want to manually dead-letter messages? To what purpose?

Reply:
Testing / mimic real world situations. Before i was only able to dead-letter a message by setting the TTL or by tweaking a consumer to return an error. This button consumes the message and then rejects it the way an application would. Just saves time and you could even use it for Unit testing RabbitMQ if you wanted to. In any case it will save me time with future requests and issues in RabbitMQ.

Reply:
I don't really see the need, but I also don't mind enough to stop the merge